pkg/cmd/urlcheck/lib/urlcheck/urlcheck.go

```diff
@@ -119,25 +122,21 @@ func checkURL(client *http.Client, url string) error {
 		return nil
 	}
 
-	if resp.StatusCode == 403 || resp.StatusCode == 405 {
-		// The server doesn't like HEAD. Try GET. This is obviously the
-		// correct strategy for 405 Method Not ALlowed responses. This is a
-		// less-correct strategy for 403 Forbidden responses, but we link to
-		// several misconfigured S3 buckets that return 403 Forbidden for HEAD
-		// requests but not for GET requests.
-		retryResp, err := client.Get(url)
-		if err != nil {
-			return err
-		}
-		if err := retryResp.Body.Close(); err != nil {
-			return err
-		}
-
-		if retryResp.StatusCode >= 200 && retryResp.StatusCode < 300 {
-			return nil
-		}
+	// The server doesn't like HEAD. Try GET. This is obviously the correct
+	// strategy for a 405 Method Not ALlowed error, but a less-correct strategy
```

Comment:
s/ALlowed/Allowed

Reply:
Eek, thanks! My editor's font makes that typo nearly invisible.